test/params_api_test.c

```diff
@@ -431,7 +431,7 @@ static int test_param_bignum(int n)
 
     if (!TEST_true(OSSL_PARAM_set_BN(&param, b)))
         goto err;
-    le_copy(buf, bnbuf, len);
+    le_copy(buf, bnbuf, sizeof(bnbuf));
```

Comment:
Memseting buf to zero also would have worked right?

Reply:
IMO no